### PR TITLE
Revert "New version 0.16.5"

### PR DESCRIPTION
**Pull request by telemaxx**
next version should be 0.16.6
Reverts telemaxx/mapsforgesrv#19

### CODE REVIEW

File: mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java [hunk 1]
```diff
@@ -22,102 +22,344 @@
...
 import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.util.thread.ExecutorThreadPool;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.slf4j.bridge.SLF4JBridgeHandler;
 
 public class MapsforgeSrv {
-	
-	private final static String VERSION = "0.16.5"; // starting with eg 0.13, the mapsforge version //$NON-NLS-1$
```
Comment (telemaxx) [inline mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java:47]: 0.16.6

File: mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java [hunk 2]
```diff
@@ -22,102 +22,344 @@
  * 0.16.2: Check for valid tile numbers (JFBeck)
  * 0.16.3: hillshading (JFBeck)
  * 0.16.4: gamma correction (JFBeck)
- * 0.16.5: a lot of improvements/rework (non0303)
  *******************************************************************************/
```
Comment (telemaxx) [inline mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java:25]: 0.16.6: bugfix when parameter is not given and default value should be used

Comment (telemaxx) [reply mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java:25]: @JFritzle : can you edit 2 lines(version 0.16.6) and recreate the jar file?
[0.16.6 ](https://github.com/telemaxx/mapsforgesrv/commit/95e5c14814a897745e5ea0f4cd4559cafbbf466f)

File: mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java (hunk 1 above)
Comment (JFritzle) [reply mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java:47]: Version and jar fils is now 0.16.6, includes bugfixes and improvements, sending a PR

**Pull request merged**